week2/Task9/Program.cs

```diff
@@ -0,0 +1,77 @@
+﻿Console.WriteLine("Hello");
+
+var jt = new JupiterTime(55,55);
+
+Console.WriteLine(jt);
+
+var tt = new TitanTime(55,55);
+
+Console.WriteLine(tt);
+
+var gt = new GanymedeTime(55,55);
+
+Console.WriteLine(gt);
+public abstract class AlienTime
+{
+    public int Hours { get; set; }
+    public int Minutes { get; set; }
+    private int HoursInDay;
+    public AlienTime(int hours, int minutes, int hoursInDay)
+    {
+        Hours = hours;
+        Minutes = minutes;
+        HoursInDay = hoursInDay;
+    }
+    public override string ToString()
+    {
+        
+        if(Minutes < 10)
+        {   if(HoursInDay < 100)
```

Comment:
You should have used `HoursInDay` to correctly display hours if large number of hours has been entered (Hours > HoursInDay).

Same applies to minutes, like in task #2 someone can enter large number of minutes eg 88.